databricks/koalas/indexes.py

```diff
@@ -2692,6 +2716,13 @@ def get_level_values(self, level):
         )
         return ks.DataFrame(internal).index
 
+    def holds_integer(self):
+        """
+        Whether the type is an integer type.
+        Always return False for MultiIndex.
+        """
+        return False
```

Comment:
Do we need this?

Reply:
Oh, I think we don't need this.
I moved the docstring to the `Index.holds_integer` and removed the `MultiIndex.holds_integer`.